specification/openapi.json

```
@@ -0,0 +1,147 @@
+{
+  "openapi": "3.0.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "Swagger Adamant-wallets"
+  },
+  "paths": {},
+  "components": {
+    "schemas": {
+      "TokenGeneral": {
+        "allOf": [
+          {
+            "$ref": "#/components/schemas/TokenGeneral"
+          },
+          {
+            "type": "object",
+            "required": [
+              "name",
+              "website",
+              "description",
+              "explorer",
+              "symbol",
+              "type",
+              "decimals",
+              "qqPrefix",
+              "status",
+              "links"
+            ],
+            "properties": {
+              "name": {
+                "type": "string"
+              },
+              "website": {
+                "type": "string"
+              },
+              "description": {
+                "type": "string"
+              },  
+              "explorer": {
+                "type": "string"
+              },
+              "symbol": {
+                "type": "string"
+              },
+              "type": {
+                "type": "string",
+                "enum": [
+                  "coin",
+                  "ERC20"
+                ]
+              },    
+              "decimals": {
+                "type": "integer"
+              },
+              "minBalance": {
+                "type": "integer"
+              },
+              "cryptoTransferDecimals": {
+                "type": "integer"
+              },
+              "minTransferAmount": {
+                "type": "integer"
+              },
+              "fixedFee": {
+                "type": "integer"
+              },  
+              "qqPrefix": {
+                "type": "string"
```

What is this prop for? It's not clear for me. Add a description maybe.

```suggestion
                "type": "string",
                "description": "Property description"
```